### ISSUE TITLE
all: perf issues

### ISSUE THREAD

**Issue by cristaloleg** [go-critic/go-critic#352]
Some of the checkers consume more time than they should(citation needed).
We might be interested into speeding them:
```bash
    --- PASS: TestCheckers/emptyFmt (0.84s)
    --- PASS: TestCheckers/flagDeref (0.66s)
    --- PASS: TestCheckers/importShadow (1.90s)
    --- PASS: TestCheckers/namedConst (0.40s)
    --- PASS: TestCheckers/rangeValCopy (0.76s)
    --- PASS: TestCheckers/regexpMust (0.59s)
```

**Comment by Quasilyte** [comment]
Maybe due to imports?
If you're talking about tests.
This is why we avoid imports in testdata whether possible.

**Comment by cristaloleg** [comment]
Yep, looks like it's due imports and most of this time was consumed by loader. Closing.
